.github/workflows/pre-main.yaml

```diff
@@ -239,31 +239,12 @@ jobs:
       - name: 'Show failed test cases and the non compliant objects'
         run: ./certsuite claim show failures -c certsuite-out/claim.json
 
-      - name: Upload smoke test results as an artifact
```

Comment:
Are there any issues with keeping these? I used these artifacts several times in the past to debug issues or obtain recent claim files.

Reply:
I didn't know anyone was still using them.  We can keep them if you want.